### DISCUSSION TITLE
Components Aura

### DISCUSSION THREAD

**Discussion by annefranca** [Telefonica/mistica-design#1332] [Proposals]
### Goal

Integrar assets da Aura (chat de atendimento da Vivo) em uma biblioteca (Mística/Extendend).

### Context

Atualmente temos um fluxo de atendimento (Aura) onde os assets foram criados atomicamente. Essa jornada de atendimento vai ser atualizada de acordo com a interface/conceito do Novo App.

### Impact

Adicionar esses assets que hoje estão em um arquivo da squad em uma biblioteca, tornaria esse compilado de componentes reutilizável e consistente, tendo em vista que outros times usam a mesma interação para o atendimento com a pessoa usuária (chatbot).

### Discussion Questions

- As outras marcas trabalham com atendimento por meio de um "chatbot"?
- Os componentes que fazem parte dessa jornada poderiam estar no Mística?

### Additional considerations

Adicionei no arquivo abaixo o link da jornada completa, os principais assets e algumas telas de interação que fazem parte desse fluxo de atendimento.

### Do you have a visual proposal?

https://www.figma.com/file/HGp27iaT2x1OxpDN7QXbgz/Components-Jornada-Aura?type=design&node-id=0%3A1&mode=design&t=grxARTgrjpNvfzDQ-1

**Comment by annefranca** [comment]
@yceballost ficou alguma dúvida sobre essa discussão?

**Reply by yceballost** [reply]
We are discussing with the Aura team how this should be included.

**Comment by yceballost** [comment]
## More Context
We have had discussions with the Aura and Vivo teams, and we believe it would be beneficial for Mística to incorporate conversational components aimed at addressing conversational interfaces.

Currently, Aura provides non-visual conversational components (a response, a suggestion, response options, Aura's avatar (which is more visual), ...) and Mística should have a series of components that ensure they align with the non-visual components provided by Aura.

Currently, if you use Aura on WhatsApp, it will use WhatsApp's components, and the same goes for Teams and other services that are ready to consume adaptive card technology. In the future, it would be ideal for Aura to incorporate these components into the WebClient so that Aura itself not only displays non-visual components but also draws from Mística and provides you with the complete package.

The Aura team believes that Aura needs to visually blend in with the product's UI, so it would be a perfect fit to have visual components from Mística that can change according to the brand.

## Next Steps
- Identify the conversational components that at a minimum cover what Aura provides
- Create a list with this components
- Align them with the roadmap.

**Reply by annefranca** [reply]
@BrunoBiagioniNeto

**Comment by yceballost** [comment]
The list that I can extract by the moment is:
- Bubbles (con diferentes tipologías)
- Message bar

I made [a branch](https://www.figma.com/file/APBal1yWWn9F1i1ZDc0Vv4/%23X---Conversational-components?type=design&node-id=0-1&mode=design) in a new spec document to define these components

I will close this ticket and I will create an issue

**Reply by annefranca** [reply]
@yceballost 

Tivemos um retorno que a equipe de Aura vai trabalhar nos elementos para a interface de Aura, pois não usam Mística, nem componentes de um DS. Eles trabalham forma hard code.

Vou alinhando com você as novidades sobre esse fluxo.

Obrigada :)
